modules/op-creating-and-verifying-task-run-signatures-without-any-additional-authentication.adoc

```diff
@@ -0,0 +1,92 @@
+    // This module is included in the following assembly:
```

Comment:
this is showing up in the docs 😨

Reply:
![Screenshot from 2022-04-06 18-05-12](https://user-images.githubusercontent.com/9133660/161975701-e34fe897-2eb7-4c45-8e09-c1cef12b5bf4.png)